Add `<ReadMore>` component (#6267)
withastro/docs: src/content/docs/en/guides/integrations-guide/markdoc.mdx

});
```

> <ReadMore>[Find all of Markdoc's built-in nodes and node attributes on their documentation.](https://markdoc.dev/docs/nodes#built-in-nodes)</ReadMore>

Reviewer: ```suggestion
<ReadMore>Find all of the built-in nodes and node attributes on the [Markdoc nodes documentation](https://markdoc.dev/docs/nodes#built-in-nodes).</ReadMore>
```

Reviewer 2: I was going to do this when I got to my laptop :P

Author: Dont love having a “their” before “Markdoc” is mentioned. Also not sure if “their” is appropriate (who is the plural group? is “Markdoc” they?)

Going to go with this instead!

```suggestion
<ReadMore>See the [Markdoc nodes documentation](https://markdoc.dev/docs/nodes#built-in-nodes) to learn about all the built-in nodes and attributes.</ReadMore>
```
